website/versioned_docs/version-27.4/JestObjectAPI.md

```diff
@@ -716,7 +716,7 @@ jest.setTimeout(1000); // 1 second
 
 ### `jest.retryTimes()`
 
-Runs failed tests n-times until they pass or until the max number of retries is exhausted. This only works with the default [jest-circus](https://github.com/facebook/jest/tree/main/packages/jest-circus) runner!
+Runs failed tests n-times until they pass or until the max number of retries is exhausted. This only works with the default [jest-circus](https://github.com/facebook/jest/tree/main/packages/jest-circus) runner!  This must live at the top-level of a test file or in a describe block. Retries will not work if `jest.retryTimes()` is called in a beforeEach block.
```

Comment:
also won't work from within a `test` block (I think? I've never used this feature 😅)